src/client/components/LeaderboardList/AmountOfTime.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AmountOfTime({ timeSpent }) {
+  const hours = Math.floor(timeSpent / 3600);
+  const minutes = Math.floor((timeSpent % 3600) / 60);
+  const seconds = Math.floor((timeSpent % 3600) % 60);
+
+  if (!timeSpent) {
```

Comment:
Try to avoid code dulication

Reply:
Thanks for feedback, i only have one question , if i changed className for parent div so duplication solved? 
return (
      <div className="avatar-failed-div">
        <div>avatar URL missing</div>
      </div>
    );
  }

  return (
    <div className="avatar-div">
      <img className="avatar" src={avatar} alt="avatar-img" />
    </div>
  );

Reply:
Thats one way of doing it, yes